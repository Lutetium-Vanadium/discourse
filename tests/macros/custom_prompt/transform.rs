fn main() {
    let q = requestty::questions![Custom { transform: todo!() }];
}
