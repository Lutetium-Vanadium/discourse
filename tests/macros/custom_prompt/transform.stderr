error: option `transform` does not exist for kind `custom`
 --> $DIR/transform.rs:2:44
  |
2 |     let q = requestty::questions![Custom { transform: todo!() }];
  |                                            ^^^^^^^^^
